Gemfile

```diff
@@ -3,7 +3,8 @@ source 'https://rubygems.org'
 gem 'rails', '4.2.5.2'
 gem 'slimmer', '10.0.0'
 
-gem 'govuk_frontend_toolkit', '~> 3.1.0'
+gem 'govuk_elements_rails', '~> 3.0.1'
```

Comment:
As far as I know you don't have or hardly have anything like this on GOV.UK.  Looks like you had a chat about it though, so that's good.